nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -1,14 +1,55 @@
 # Release 22.11 (“Raccoon”, 2022.11/??) {#sec-release-22.11}
 
-Support is planned until the end of June 2023, handing over to 23.05.
+This release is supported until the end of June 2023, handing over to NixOS 23.05.
 
 ## Highlights {#sec-release-22.11-highlights}
 
-In addition to numerous new and upgraded packages, this release has the following highlights:
+In addition to numerous new and upgraded packages, this release includes the following highlights:
 
 - GNOME has been upgraded to 43. Please take a look at their [Release
   Notes](https://release.gnome.org/43/) for details.
 
+- KDE Plasma has been upgraded to 5.25 and 5.26.
+    - https://kde.org/announcements/plasma/5/5.25.0/
+    - https://kde.org/announcements/plasma/5/5.26.0/
+
+- Cinnamon has been updated to 5.4 and the Cinnamon module now defaults to
+  blueman as Bluetooth manager and slick-greeter as LightDM greeter to match upstream.
+
+- OpenSSL now defaults to OpenSSL 3, updated from 1.1.1.
+
+- PHP now defaults to PHP 8.1, updated from 8.0.
+
+- PHP is now built `NTS` (Non-Thread Safe) style by default, for Apache and
+  `mod_php` usage we still enable `ZTS` (Zend Thread Safe). This has been a
+  common practice for a long time in other distributions.
+
+- PHP 8.2.0 RC 7 is available.
```

Comment:
I think we just need one paragraph for php. Also agree that we can drop the release candidate.

Reply:
I think the version bump and thread mode stuff are fine separate. Don't think it matters too much, though. Thoughts?